### PR TITLE
Update README

### LINKED ISSUE #585: using bootstrap through proxy on protected hosts
I'm trying to use a proxy to bootstrap a protected host.

```
# On proxy host
yum install tinyproxy
vi /etc/tinyproxy/tinyproxy.conf
#> Allow protected.system
service tinyproxy start

# On protected system
export http_proxy=proxyhost.name:8888
export https_proxy=proxyhost.name:8888
curl -L http://bootstrap.saltstack.org | sudo sh -s -- -M -N git 2015.2
```

all seemed well until:

```
Nothing to do
 *  WARN: The git revision being installed does not match a Salt version tag. Shallow cloning disabled
fatal: unable to connect to github.com:
github.com[0: 192.30.252.131]: errno=No route to host

Cloning into 'salt'...
 * ERROR: Failed to run install_centos_git_deps()!!!
```

It would be nice if bootstrap could have support for this scenario.  If it already does then a little help in the README would be a nice touch.

### PR DESCRIPTION
**Pull request by vutny**
- Fix some formatting and trailing whitespaces
- Actualized supported OS list
- Add explanation how to use HTTP proxy, resolves #585 

Ping @s0undt3ch

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,139 +2,159 @@
...
-  sudo sh install_salt.sh
+  wget -O bootstrap_salt.sh https://bootstrap.saltstack.com
+  sudo sh bootstrap_salt.sh -P git v2015.8.7
 
-If you have any SSL issues install ``ca_root_nssp``:
+.. note::
 
-.. code:: console
+  On the above example we added `-P` which will allow PIP packages to be installed if required but
+  it's no a necessary flag for Git based bootstraps.
```
Comment (s0undt3ch) [inline README.rst]: not?

File: README.rst [hunk 2]
```diff
@@ -2,139 +2,159 @@
...
 
 Examples
-~~~~~~~~
+--------
 
 .. note::
 
-    In every two-step example, you would be well-served to examine the downloaded file and examine
-    it to ensure that it does what you expect.
+  Those examples below show how to bootstrap Salt directly from GitHub or other Git repository.
```
Comment (s0undt3ch) [inline README.rst]: These examples?

**Comment by vutny** [comment]
@s0undt3ch Thank you for careful reading! I'll correct that.

**Comment by vutny** [comment]
@s0undt3ch 
Fixed.

**Comment by vutny** [comment]
@s0undt3ch Also I have plans to update [Bootstrap tutorial](https://docs.saltstack.com/en/latest/topics/tutorials/salt_bootstrap.html) page in Salt main docs to make it more consistent with README here.
Also, that should partially resolve #782 in matter of documentation.

Looking forward to have this PR merged. Thank you!

**Comment by s0undt3ch** [comment]
Thank You!

**Pull request merged**